src/ui/Controllers/HomeController.cs

```diff
@@ -20,7 +20,7 @@ public HomeController(ManageApi manageApi)
         public ActionResult Index()
         {
 
-            return this.RedirectToAction("imported");
```

Comment:
We should also probably remove the `Imported.cshtml` if we're no longer linking to it.